containers/server-image/Dockerfile

```diff
@@ -50,5 +49,26 @@ LABEL org.openbuildservice.disturl="%DISTURL%"
 LABEL org.opensuse.reference="${REFERENCE_PREFIX}/server:%PKG_VERSION%.%RELEASE%"
 # endlabelprefix
 
+EXPOSE 443/tcp
+EXPOSE 80/tcp
+# Salt
+EXPOSE 4505/tcp
+EXPOSE 4506/tcp
+# PXE server
+EXPOSE 69/udp
+# Cobbler
+EXPOSE 25151/tcp
+# Postgres for report db
+EXPOSE 5432/tcp
+# HTTP ssh tunnel
+#EXPOSE 1232/tcp
+# HTTPS ssh tunnel
+#EXPOSE 1233/tcp
+#TODO these ports should be exposed in debug mode
+# Tomcat remote debug
+#EXPOSE 8000/tcp
+# Taskomatic remote debug
+#EXPOSE 8001/tcp
+
```

Comment:
Just remove all ports since those will be defined outside anyway: no need to duplicate these data